src/lake/Lake/Build/Facets.lean

```diff
@@ -74,7 +74,19 @@ module_data ilean : BuildJob FilePath
 abbrev Module.cFacet := `c
 module_data c : BuildJob FilePath
 
-/-- The object file built from `c`. -/
+/-- The LLVM BC file built from the Lean file via `lean`. -/
+abbrev Module.bcFacet := `bc
+module_data bc : BuildJob FilePath
+
+/-- The object file `.c.o` built from `c`. -/
+abbrev Module.cOFacet := `cO
```

Comment:
The name of this facet should be `` `c.o `` (and `` `bc.o `` for the bc facet). I would also suggested lower-casing the `O` in the definition name (i.e., `coFacet` seems more elegant), but I am less confident on that stylistic decision.